### PR TITLE
(feat)docs: add kafka broker kill docs

### PR DESCRIPTION
**Pull request by ksatchit**
- Add kafka as a sidebar item
- Add kafka broker kill experiment docs

Signed-off-by: ksatchit <ksatchit@mayadata.io>

### CODE REVIEW

File: docs/kafka-broker-pod-failure.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+sidebar_label: Kafka Broker Pod Failure
+---
+
+## Experiment Metadata
+
+| Type  | Description                    | Kafka Distribution   | Tested K8s Platform
+| ----- | -------------------------------|----------------------|---------------------
+| Kafka | Fail kafka leader-broker pods  | Confluent, Kudo-Kafka| AWS Konvoy, GKE
+
+## Pre-Requisites
```
Comment (chandankumar4) [inline docs/kafka-broker-pod-failure.md]: The `-` is not looking good, It should be Prerequisite

File: docs/kafka-broker-pod-failure.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+- If Confluent/Kudo Operators have been used to deploy Kafka, note the instance name, which will be 
+  used as the value of `KAFKA_INSTANCE_NAME` experiment environment variable 
+
+  - In case of Confluent, specified by the `--name` flag
+  - In case of Kudo, specified by the `--instance` flag
+ 
+  Zookeeper uses this to construct a path in which kafka cluster data is stored.  
+
+
+## Entry-Criteria
```
Comment (chandankumar4) [inline docs/kafka-broker-pod-failure.md]: Here also **Entry Criteria**

File: docs/kafka-broker-pod-failure.md [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+
+- Kafka Cluster (comprising the Kafka-broker & Zookeeper Statefulsets) is healthy
+
+## Exit-Criteria
```
Comment (chandankumar4) [inline docs/kafka-broker-pod-failure.md]: **Exit Criteria**

**Review by umamukkara — DISMISSED** [review]
LGTM

File: docs/kafka-broker-pod-failure.md [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+## Details
+
+- Causes (forced/graceful) pod failure of specific/random Kafka broker pods
+- Tests deployment sanity (replica availability & uninterrupted service) and recovery workflows of the Kafka cluster
+- Tests unbroken message stream when `KAFKA_LIVENESS_STREAM` experiment environment variable is set to `enabled`
+
+## Integrations
+
+- Pod failures can be effected using one of these chaos libraries: `litmus`, `powerfulseal`
+- The desired chaoslib can be selected by setting one of the above options as value for the environment variable `LIB`
```
Comment (chandankumar4) [inline docs/kafka-broker-pod-failure.md]: `chaoslib` -> `chaos library`

File: docs/kafka-broker-pod-failure.md [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+
+- This Chaos Experiment can be triggered by creating a ChaosEngine resource on the cluster.
+  To understand the values to provide in a ChaosEngine specification, refer [Getting Started](getstarted.md/#prepare-chaosengine) 
+
+- Follow the steps in the sections below to prepare the ChaosEngine & execute the experiment.
+
+### Prepare ChaosEngine
+
+- Provide the application info in `spec.appinfo`
+- Provide the experiment tunables. While many tunables have defaults specified in the ChaosExperiment CR, some need to be
```
Comment (chandankumar4) [inline docs/kafka-broker-pod-failure.md]: `tunables.` -> `tunables,`
`While many tunables have defaults specified` -> `While many tunables have defaults value specified`

**Comment by ksatchit** [comment]
Thanks @chandankumar4 -- made the changes.

**Review by chandankumar4 — APPROVED** [review]
LGTM

**Pull request merged**